storage/src/main/scala/uk/ac/wellcome/storage/store/RetryableReadable.scala

```diff
@@ -33,7 +27,10 @@ trait RetryableGet[T] {
       Try {
         retryableGetFunction(location)
       } match {
-        case Success(t)   => Right(t)
-        case Failure(err) => Left(buildGetError(err))
+        case Success(t) => Right(t)
+        case Failure(err) =>
+          val error = buildGetError(err)
+          warn(s"$error when trying to get $location")
```

Comment:
Let's swap this around slightly:

```suggestion
          warn(s"Error when trying to get $location: $error")
```

The error could be much longer than the location. Having them all neatly line up will make it easier to spot commonalities if you have a stack of logs next to each other.